app/controllers/api/v1/tags_controller.rb

```diff
@@ -6,12 +6,14 @@ class TagsController < ApplicationController
 
       def index
         if params[:portfolio_id]
+          Portfolio.find(params.require(:portfolio_id))
```

Comment:
How much information do we need to return to the end user? Is it OK if we just do something like:
```ruby
return head :not_found unless Portfolio.where(params.require(:portfolio_id)).exists?
```
If not, can we somehow incorporate `.exists?`

I don't want to have to look up the same thing twice if it exists, might as well just do an `exists?` check on the record if we can for at least a little bit of performance.